bundles/pom.xml

```diff
@@ -1,364 +1,711 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you check what happpened in this file?

Reply:
No idea what happened - fixed in 4714fc9b7d